### ISSUE TITLE
performance  about LIP Dataset

### ISSUE THREAD

**Issue by DerrickWang005** [HRNet/HRNet-Semantic-Segmentation#112]
Hello, I evaluated HRNetV2 with your released pretrained model on LIP Dataset. And, I didn't change anything in .yaml file. But, I got the result of mIoU=54.08. So, could you please help me to fix this problem?

**Comment by sunke123** [comment]
If you using the pytorch-v1.1，you should download the released model on pytorch-v1.1.
The batchnorm is not the same between our pytorch-v1.1 and v0.4.1 braches.
